lib/views/help/officers.html.erb

```diff
@@ -2,6 +2,14 @@
 <%= render :partial => 'sidebar' %>
 <div id="left_column_flip" class="left_column_flip">
   <h1 id="officers"><%= @title %> <a href="#officers">#</a> </h1>
+  <div class="article-grid__item with-background">
```

Comment:
This looks great, but would position at the bottom of the page

![Screenshot 2023-06-28 at 15 10 08](https://github.com/mysociety/whatdotheyknow-theme/assets/282788/cb9c376c-99e6-4f14-b5b3-8c81ea234131)